sway-lib-std/src/u128.sw

```diff
@@ -299,6 +299,7 @@ impl U128 {
         }
     }
 
+    // TODO: Rename to `try_as_u64` to be consistent with all other downcasts
```

Comment:
Can an issue be created for this?

Reply:
https://github.com/FuelLabs/sway/issues/6954